week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,17 +5,26 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(Arr) {
+let String=Arr.map(tidy=>tidy.trim().toLowerCase().replace("/",""));
 
+return String; 
+}
```

Comment:
Tip:
 * as a variable naming convention use camelCase naming format, it is a defacto stander
   https://www.freecodecamp.org/news/javascript-naming-conventions-dos-and-don-ts-99c0e2fdd78a/
* Choose meaningful names for variables which tells more information what is stored, what the aim is.
   It helps to reason about the code.
* Pay more attention to code formatting, especially in this case using neat indentation that help developers to understand your code faster.